riscv/encoding.h

```diff
@@ -1844,6 +1844,673 @@
+#define MATCH_UKADDW 0x1000107f
+#define MASK_UKADDW 0xfe00707f
+#define MATCH_UKCRAS16 0x3400007f
+#define MASK_UKCRAS16 0xfe00707f
+#define MATCH_UKCRSA16 0x3600007f
+#define MASK_UKCRSA16 0xfe00707f
+#define MATCH_UKMAR64 0xb400107f
+#define MASK_UKMAR64 0xfe00707f
+#define MATCH_UKMSR64 0xb600107f
+#define MASK_UKMSR64 0xfe00707f
+#define MATCH_UKSTAS16 0x3400307f
+#define MASK_UKSTAS16 0xfe00707f
+#define MATCH_UKSTSA16 0x3600307f
+#define MASK_UKSTSA16 0xfe00707f
+#define MATCH_UKSUB8 0x3a00007f
+#define MASK_UKSUB8 0xfe00707f
+#define MATCH_UKSUB16 0x3200007f
+#define MASK_UKSUB16 0xfe00707f
+#define MATCH_UKSUB64 0xb200107f
+#define MASK_UKSUB64 0xfe00707f
+#define MATCH_UKSUBH 0x1600107f
+#define MASK_UKSUBH 0xfe00707f
+#define MATCH_UKSUBW 0x1200107f
+#define MASK_UKSUBW 0xfe00707f
+#define MATCH_UMAR64 0xa400107f
+#define MASK_UMAR64 0xfe00707f
+#define MATCH_UMAQA 0xcc00007f
+#define MASK_UMAQA 0xfe00707f
+#define MATCH_UMAX8 0x9a00007f
+#define MASK_UMAX8 0xfe00707f
+#define MATCH_UMAX16 0x9200007f
+#define MASK_UMAX16 0xfe00707f
+#define MATCH_UMIN8 0x9800007f
+#define MASK_UMIN8 0xfe00707f
+#define MATCH_UMIN16 0x9000007f
+#define MASK_UMIN16 0xfe00707f
+#define MATCH_UMSR64 0xa600107f
+#define MASK_UMSR64 0xfe00707f
+#define MATCH_UMUL8 0xb800007f
+#define MASK_UMUL8 0xfe00707f
+#define MATCH_UMULX8 0xba00007f
+#define MASK_UMULX8 0xfe00707f
+#define MATCH_UMUL16 0xb000007f
+#define MASK_UMUL16 0xfe00707f
+#define MATCH_UMULX16 0xb200007f
+#define MASK_UMULX16 0xfe00707f
+#define MATCH_URADD8 0x2800007f
+#define MASK_URADD8 0xfe00707f
+#define MATCH_URADD16 0x2000007f
+#define MASK_URADD16 0xfe00707f
+#define MATCH_URADD64 0xa000107f
+#define MASK_URADD64 0xfe00707f
+#define MATCH_URADDW 0x3000107f
+#define MASK_URADDW 0xfe00707f
+#define MATCH_URCRAS16 0x2400007f
+#define MASK_URCRAS16 0xfe00707f
+#define MATCH_URCRSA16 0x2600007f
+#define MASK_URCRSA16 0xfe00707f
+#define MATCH_URSTAS16 0x2400307f
+#define MASK_URSTAS16 0xfe00707f
+#define MATCH_URSTSA16 0x2600307f
+#define MASK_URSTSA16 0xfe00707f
+#define MATCH_URSUB8 0x2a00007f
+#define MASK_URSUB8 0xfe00707f
+#define MATCH_URSUB16 0x2200007f
+#define MASK_URSUB16 0xfe00707f
+#define MATCH_URSUB64 0xa200107f
+#define MASK_URSUB64 0xfe00707f
+#define MATCH_URSUBW 0x3200107f
+#define MASK_URSUBW 0xfe00707f
+#define MATCH_WEXTI 0xde00007f
+#define MASK_WEXTI 0xfe00707f
+#define MATCH_WEXT 0xce00007f
+#define MASK_WEXT 0xfe00707f
+#define MATCH_ZUNPKD810 0xacc0007f
+#define MASK_ZUNPKD810 0xfff0707f
+#define MATCH_ZUNPKD820 0xacd0007f
+#define MASK_ZUNPKD820 0xfff0707f
+#define MATCH_ZUNPKD830 0xace0007f
+#define MASK_ZUNPKD830 0xfff0707f
+#define MATCH_ZUNPKD831 0xacf0007f
+#define MASK_ZUNPKD831 0xfff0707f
+#define MATCH_ZUNPKD832 0xad70007f
+#define MASK_ZUNPKD832 0xfff0707f
+#define MATCH_RDOV 0x80102073
+#define MASK_RDOV 0xfffff07f
+#define MATCH_CLROV 0x8010f073
+#define MASK_CLROV 0xffffffff
+/* NDS V5 DSP Extension (RV64 only).  */
+#define MATCH_ADD32 0x4000207f
```

Comment:
Is there only opcode but no real implementation.  If it is yes, remove it and make it live with 32 instruction in another PR.

Reply:
OK. I will remove the opcode that is not implemented now.